.gitignore

```diff
@@ -48,6 +48,7 @@ doxygen.cfg
 src/wpantund/wpantund
 src/wpanctl/wpanctl
 .dirstamp
+src/version.c
```

Comment:
Thanks for your PR. I see the file at `src/wpantund/version.c`?

Reply:
Ah, I think this is caused by the difference between Android.mk and src/Makefile.am.
Android.mk generates src/version.c on line 75 where as src/Makefile.am generates  $(top_builddir)/$(subdir)/version.c on line 47. I can update the PR to include both locations.

Reply:
Sounds good. Thanks.